### PR TITLE
Update `Tournaments/4DM/2024`

### PR DESCRIPTION
**Pull request by Albionthegreat**
Additional Rules for Team Formation have been added as well as the skillban list. Staff Listing is up to date again.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/4DM/2024/en.md [hunk 1]
```diff
@@ -43,9 +43,9 @@ The 4 Digit osu!mania World Cup 2024 is run by various community members.
...
+| Streamer | ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595), ::{ flag=US }:: [EpsilonMaiagare](https://osu.ppy.sh/users/3855052), ::{ flag=SE }:: [Logg45vs ](https://osu.ppy.sh/users/8684540), ::{ flag=BE }:: [Mortelspawn_](https://osu.ppy.sh/users/5331420), ::{ flag=NL }:: [NightNarumi](https://osu.ppy.sh/users/4381142), ::{ flag=US }:: [SunApple](https://osu.ppy.sh/users/11817622), ::{ flag=US }:: [Znow](https://osu.ppy.sh/users/15513303) |
```
Comment (Walavouchey) [inline wiki/Tournaments/4DM/2024/en.md:47]:
```suggestion
| Streamer | ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595), ::{ flag=US }:: [EpsilonMaiagare](https://osu.ppy.sh/users/3855052), ::{ flag=SE }:: [Logg45vs](https://osu.ppy.sh/users/8684540), ::{ flag=BE }:: [Mortelspawn_](https://osu.ppy.sh/users/5331420), ::{ flag=NL }:: [NightNarumi](https://osu.ppy.sh/users/4381142), ::{ flag=US }:: [SunApple](https://osu.ppy.sh/users/11817622), ::{ flag=US }:: [Znow](https://osu.ppy.sh/users/15513303) |
```

File: wiki/Tournaments/4DM/2024/en.md [hunk 2]
```diff
@@ -73,15 +73,15 @@ The 4 Digit osu!mania World Cup 2024 is run by various community members.
    2. ...not have violated the [osu! community rules](/wiki/Rules) in the past 12 months.
    3. ...not exceed the global osu!mania rank of #250.
    4. ...not have been part of a former winning 4DM team. (4DM1 winners are excluded from this.)
-   5. ...not appear on the skillbans list. Skillbans will be announced on the 1st of December 2023.
+   5. ...not appear on the [skillban list](https://docs.google.com/spreadsheets/d/1wQ_6u-uUYhfeVujY0GERB-bWa0fi78KmW9rqeJHrTII).
```
Comment (Walavouchey) [inline wiki/Tournaments/4DM/2024/en.md:76]:
```suggestion
   5. ...not appear on the [skillban list](https://docs.google.com/spreadsheets/d/1wQ_6u-uUYhfeVujY0GERB-bWa0fi78KmW9rqeJHrTII).
```

File: wiki/Tournaments/4DM/2024/en.md [hunk 3]
```diff
@@ -73,15 +73,15 @@ The 4 Digit osu!mania World Cup 2024 is run by various community members.
...
-2. A [skill-based restriction system](https://4digitmwc.github.io/skillban-articles) developed and maintained by ::{ flag=TH }:: [HowToPlayLN](https://osu.ppy.sh/users/10879600), and ::{ flag=SG }:: [Polytetral](https://osu.ppy.sh/users/8612061) will be in effect in order to keep 4DM2024 fair to all players and as free of malicious sandbagging as possible. To see who is considered to be too skilled for the tournament by this system, see the skillban list (coming December 1st 2023).
+2. A [skill-based restriction system](https://4digitmwc.github.io/skillban-articles) developed and maintained by ::{ flag=TH }:: [HowToPlayLN](https://osu.ppy.sh/users/10879600), and ::{ flag=SG }:: [Polytetral](https://osu.ppy.sh/users/8612061) will be in effect in order to keep 4DM2024 fair to all players and as free of malicious sandbagging as possible. To see who is considered to be too skilled for the tournament by this system, see the [skillban list](https://docs.google.com/spreadsheets/d/1wQ_6u-uUYhfeVujY0GERB-bWa0fi78KmW9rqeJHrTII). For more information on how the system works, please check the [github repository](https://github.com/4digitmwc/skillbans-in-2024).
```
Comment (Walavouchey) [inline wiki/Tournaments/4DM/2024/en.md:84]:
```suggestion
2. A [skill-based restriction system](https://4digitmwc.github.io/skillban-articles) developed and maintained by ::{ flag=TH }:: [HowToPlayLN](https://osu.ppy.sh/users/10879600), and ::{ flag=SG }:: [Polytetral](https://osu.ppy.sh/users/8612061) will be in effect in order to keep 4DM2024 fair to all players and as free of malicious sandbagging as possible. To see who is considered to be too skilled for the tournament by this system, see the [skillban list](https://docs.google.com/spreadsheets/d/1wQ_6u-uUYhfeVujY0GERB-bWa0fi78KmW9rqeJHrTII). For more information on how the system works, please check the [GitHub repository](https://github.com/4digitmwc/skillbans-in-2024).
```

File: wiki/Tournaments/4DM/2024/en.md [hunk 4]
```diff
@@ -91,6 +91,9 @@ The 4 Digit osu!mania World Cup 2024 is run by various community members.
    - At the end of the registration and screening phases, a team captain will be picked by the hosts from each country's signup list based on tournament competency.
    - This team captain will pick the members to play alongside them on their country's team. The deadline for captains to submit their countries' teams is by January 5th at 23:59 UTC.
    - Teams will consist of a minimum of 3 members, and a maximum of 6 members per team.
+2. If a country or territory is unable to field enough players to make a full team (a team consisting of 6 players) even if all eligible players from that country or territory sign up, they may be able to play for, or play with, other countries.
```
Comment (Walavouchey) [inline wiki/Tournaments/4DM/2024/en.md:94]:
```suggestion
2. If a country or territory is unable to field enough players to make a full team (a team consisting of 6 players) even if all eligible players from the country or territory sign up, they may be able to play for, or play with, other countries.
```

File: wiki/Tournaments/4DM/2024/en.md [hunk 5]
```diff
@@ -91,6 +91,9 @@ The 4 Digit osu!mania World Cup 2024 is run by various community members.
...
+   - Players from dependent territories (as seen on their profile flag) may choose to play for the country they are dependent on. Refer to [this article](https://wikipedia.org/wiki/Dependent_territory#Lists_of_dependent_territories) for an overview of dependeent territories.
```
Comment (Walavouchey) [inline wiki/Tournaments/4DM/2024/en.md:95]:
```suggestion
   - Players from dependent territories (as seen on their profile flag) may choose to play for the country they are dependent on. Refer to [this article](https://wikipedia.org/wiki/Dependent_territory#Lists_of_dependent_territories) for an overview of dependent territories.
```

File: wiki/Tournaments/4DM/2024/en.md (hunk 1 above)
Comment (agatemosu) [inline wiki/Tournaments/4DM/2024/en.md:47]:
```suggestion
| Referee | ::{ flag=CN }:: [\[GB\]Mafufu](https://osu.ppy.sh/users/10884561), ::{ flag=CN }:: [\[GB\]Reisen](https://osu.ppy.sh/users/8586018), ::{ flag=CN }:: [\[GB\]Rush_FTK](https://osu.ppy.sh/users/3046856), ::{ flag=US }:: [Akace100](https://osu.ppy.sh/users/9308128), ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595), ::{ flag=MY }:: [Auxesiaa](https://osu.ppy.sh/users/16417718), ::{ flag=US }:: [EpsilonMaiagare](https://osu.ppy.sh/users/3855052), ::{ flag=PE }:: [ERA Xuste](https://osu.ppy.sh/users/17989444), ::{ flag=PH }:: [Gerwin13](https://osu.ppy.sh/users/15776185), ::{ flag=IT }:: [Kiraz](https://osu.ppy.sh/users/3807675), ::{ flag=ID }:: [Kurami_San](https://osu.ppy.sh/users/8867495), ::{ flag=SE }:: [Logg45vs](https://osu.ppy.sh/users/8684540), ::{ flag=VN }:: [MashedPotato](https://osu.ppy.sh/users/10494860), ::{ flag=BE }:: [Mortelspawn_](https://osu.ppy.sh/users/5331420), ::{ flag=PH }:: [Normiplier](https://osu.ppy.sh/users/10069850), ::{ flag=BR }:: [Nukrid](https://osu.ppy.sh/users/2307484), ::{ flag=MY }:: [Onlinee](https://osu.ppy.sh/users/13630137), ::{ flag=VN }:: [Poity](https://osu.ppy.sh/users/17148657), ::{ flag=CN }:: [shizehao](https://osu.ppy.sh/users/4928674), ::{ flag=PH }:: [Silhoueska Elze](https://osu.ppy.sh/users/11517895/), ::{ flag=US }:: [SunApple](https://osu.ppy.sh/users/11817622), ::{ flag=DE }:: [TheHunter1](https://osu.ppy.sh/users/6496016), ::{ flag=US }:: [vleth](https://osu.ppy.sh/users/18556796), ::{ flag=CA }:: [walmart5193](https://osu.ppy.sh/users/16468962), ::{ flag=US }:: [Znow](https://osu.ppy.sh/users/15513303) |
| Streamer | ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595), ::{ flag=US }:: [EpsilonMaiagare](https://osu.ppy.sh/users/3855052), ::{ flag=SE }:: [Logg45vs](https://osu.ppy.sh/users/8684540), ::{ flag=BE }:: [Mortelspawn_](https://osu.ppy.sh/users/5331420), ::{ flag=NL }:: [NightNarumi](https://osu.ppy.sh/users/4381142), ::{ flag=US }:: [SunApple](https://osu.ppy.sh/users/11817622), ::{ flag=US }:: [Znow](https://osu.ppy.sh/users/15513303) |
```

File: wiki/Tournaments/4DM/2024/en.md (hunk 2 above)
Comment (agatemosu) [inline wiki/Tournaments/4DM/2024/en.md:76]:
```suggestion
   5. ...not appear on the [skillban list](https://docs.google.com/spreadsheets/d/1wQ_6u-uUYhfeVujY0GERB-bWa0fi78KmW9rqeJHrTII).
```

File: wiki/Tournaments/4DM/2024/en.md [hunk 6]
```diff
@@ -91,6 +91,9 @@ The 4 Digit osu!mania World Cup 2024 is run by various community members.
...
+   - Players from independent countries may join ranks with other independent countries, if and only if those other countries are also unable to field a full team, and if both countries lie in the same geographic region. e.g. they both lie in the Caribbean.
```
Comment (Walavouchey) [inline wiki/Tournaments/4DM/2024/en.md:96]: "geographic region" isn't a well-defined term but ok

```suggestion
   - Players from independent countries may join ranks with other independent countries, if and only if the other countries are also unable to field a full team, and if both countries lie in the same geographic region. e.g. they both lie in the Caribbean.
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**